src/components/Admin/Adminpage.tsx

```diff
@@ -26,9 +26,9 @@ const Adminpage = () => {
     }
     const { data, isLoading } = useGetUserByIdQuery(userId);
     let users = data;
-    dispatch(setUsers(users))
+    dispatch(setUsers(users));
   }
-  SetAdminRedux();
+  // SetAdminRedux();
```

Comment:
why comment?

Reply:
Removed commented code. No longer needs to be called on page load.